app/forms/claim/mentor_training_form.rb

```diff
@@ -0,0 +1,80 @@
+class Claim::MentorTrainingForm < ApplicationForm
+  DEFAULT_HOURS_COMPLETED = %w[6 20].freeze
+  attr_accessor :claim, :mentor_training, :hours_completed, :custom_hours_completed
+
+  validate :hours_completed_presence
```

Comment:
It looks like you need to add a few pieces of custom logic for this validation. Could you do?

```suggestion
  validates :hours_completed, presence: true, unless: :custom_hours_completed?
  validates :custom_hours_completed, presence: true, unless: :hours_completed?
```

Reply:
Good idea but this needs to be 
```
  validates :hours_completed, presence: true, unless: :custom_hours?
  validates :custom_hours_completed, presence: true, if: :custom_hours?